tests/config_test.go

```diff
@@ -495,17 +500,20 @@ var _ = Describe("[Serial][rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][le
 				Expect(err).ToNot(HaveOccurred())
 				defer expecter.Close()
 
-				res, err := expecter.ExpectBatch([]expect.Batcher{
+				res, err := tests.ExpectBatchWithValidatedSend(expecter, []expect.Batcher{
 					// mount iso Secret image
 					&expect.BSnd{S: "sudo su -\n"},
-					&expect.BExp{R: "\\#"},
+					&expect.BExp{R: tests.PromptExpression},
 					&expect.BSnd{S: "mount /dev/sda /mnt\n"},
+					&expect.BExp{R: tests.PromptExpression},
 					&expect.BSnd{S: "echo $?\n"},
 					&expect.BExp{R: tests.RetValue("0")},
-					&expect.BSnd{S: "grep \"PRIVATE KEY\" /mnt/ssh-privatekey\n"},
+					&expect.BSnd{S: "grep -c \"PRIVATE KEY\" /mnt/ssh-privatekey\n"},
```

Comment:
Since the safe expecter [does not allow two BSends in a row](https://github.com/kubevirt/kubevirt/blob/master/tests/expecter.go#L210), I had to manipulate the tests and expect something after the grep command. I could have waited for prompt but I thought this will be more accurate to make it return a number greater than zero. 

I will add a note about it in the commit message.